### DISCUSSION TITLE
Contacting the owner of a GitHub organization

### DISCUSSION THREAD

**Discussion by developomp** [community/community#55400] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, I'm trying to reach out to the owner of [this GitHub Organization](https://github.com/ProjectJune), but I'm struggling to find any relevant information. Is it okay to ask for email or username of the owner?

Related discussion(s):
- #22104

**Comment by fatimasmart** [comment]
im trying to submit something through github and its not letting me copy the thing when i pressed commit changes it says ''This page isn’t working right now'' ill also send a picture of it
<img width="1918" height="1016" alt="Screenshot 2025-08-07 140726" src="https://github.com/user-attachments/assets/b52e573d-ecc9-4159-9831-453f34429556" />

**Comment by fatimasmart** [comment]
hello i want to talk to the site owner i was normally opening github and i wanted to add a file and and when i pressed commit changes it said "This page isn’t working right now
If the problem continues, contact the site owner.
HTTP ERROR 400'' and i have to send it today so can anyone knows how to fix this ill even put a picture
Screenshot 2025-08-07 140726
so please can you write an reply
fatima
